code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -120,35 +120,64 @@
 /obj/item/handcuffs/cable
 	name = "cable restraints"
 	desc = "Looks like some cables tied together. Could be used to tie something up."
-	icon_state = "cuff_white"
+	icon_state = "cablecuff"
+	item_icons = list(
+		slot_l_hand_str = 'icons/mob/items/stacks/lefthand_materials.dmi',
+		slot_r_hand_str = 'icons/mob/items/stacks/righthand_materials.dmi',
+		)
 	breakouttime = 300 //Deciseconds = 30s
 	cuff_sound = 'sound/weapons/cablecuff.ogg'
 	cuff_type = "cable restraints"
-	elastic = 1
-
-/obj/item/handcuffs/cable/red
-	color = "#DD0000"
+	elastic = TRUE
+	var/our_color
+	var/build_from_parts = TRUE
+	var/list/possible_cablecuff_colours = list(
+		"Yellow" = COLOR_YELLOW,
+		"Green" = COLOR_LIME,
+		"Pink" = COLOR_PINK,
+		"Blue" = COLOR_BLUE,
+		"Orange" = COLOR_ORANGE,
+		"Cyan" = COLOR_CYAN,
+		"Red" = COLOR_RED,
+		"White" = COLOR_WHITE
+	)
+
+/obj/item/handcuffs/cable/Initialize()
+	. = ..()
+	update_icon()
+
+/obj/item/handcuffs/cable/update_icon()
+	if(build_from_parts) //random colors!
+		if(!our_color)
+			our_color = pick(possible_cablecuff_colours)
+		var/color_hex = possible_cablecuff_colours[our_color]
+		color = color_hex
+		item_state = "coil-[our_color]"  // hardcoded. sucks, but inhands are hard and I can't be bothered.
+		add_overlay(overlay_image(icon, "[initial(icon_state)]_end", flags=RESET_COLOR))
```

Comment:
This is starting to get copypasted a bit too much for my liking

Reply:
I feel like it's out of my skill bracket to streamline this any further